firehose-source/src/main/java/source/FirehoseProperties.java

```diff
@@ -0,0 +1,118 @@
+/*
+ *  Copyright 2015 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package source;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Vinicius Carvalho
+ */
+@ConfigurationProperties(prefix = "firehose")
```

Comment:
Although we may revisit, the current "best-practice" is to not use a prefix for principal module configuration properties (_i.e._ `--dopplerUrl=foo` rather than `--firehose.dopplerUrl=foo`)